lib/manager/regex/index.ts

```diff
@@ -182,8 +157,10 @@ function handleRecursive(
     // if we have a depName and a currentValue with have the minimal viable definition
```

Comment:
`with have the` is not valid grammar, should probably be changed to:

```suggestion
    // if we have a depName and a currentValue which have the minimal viable definition
```